src/main/java/com/github/packageurl/PackageURL.java

```diff
@@ -30,6 +30,7 @@
 import java.nio.charset.StandardCharsets;
```

Comment:
Commit message nits: "... methods" (plural)

Also, if you say these methods have been generated, it would be nice to explain in the commit message body by which means they were generated. (E.g. "As generated by IntelliJ IDEA.")

Reply:
Done